h/static/scripts/streamer.js

```diff
@@ -148,12 +98,78 @@ function connect($rootScope, annotationMapper, groups, session, settings) {
         console.warn('received unsupported notification', message.type);
       }
     });
-  });
+  }
 
-  return controls;
+  function sendClientConfig () {
+    Object.keys(configMessages).forEach(function (key) {
+      if (configMessages[key]) {
+        socket.send(configMessages[key]);
+      }
+    });
+  }
+
+  /**
+   * Send a configuration message to the push notification service.
+   * Each message is associated with a key, which is used to re-send
+   * configuration data to the server in the event of a reconnection.
+   */
+  function setConfig(key, configMessage) {
+    configMessages[key] = configMessage;
+    if (socket && socket.isConnected()) {
+      socket.send(configMessage);
+    }
+  }
+
+  var connect = function connect () {
```

Comment:
You can omit the function name here, so just `var connect = function () { }`. I know that for arrow functions JS engines will infer it automatically if you do `var foo = (args) => { ... }` and I'm pretty sure they should for traditional JS functions.